config/vlan.py

```diff
@@ -97,6 +101,29 @@ def delete_db_entry(entry_name, db_connector, db_name):
         db_connector.delete(db_name, entry_name)
 
 
+def enable_stp_on_port(db, port):
+    if stp.is_global_stp_enabled(db) is True:
```

Comment:
Consider early return to reduce nesting https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
Addressed